index.html

```diff
@@ -19,9 +19,11 @@ <h1>Quiz App</h1>
     <main class="app-grid__main">
       <section class="card">
         <div class="bookmark">
-          <a href=""> <i class="fas fa-bookmark"></i> </a>
+          <a href="#">
+            <i onclick="myBookmark(this)" class="fa fa-bookmark"></i>
```

Comment:
We don't use JavaScript as yet, so you could get rid of the `onclick`-attribute here. 🙂

Reply:
That was a try to make the bookmark to toogle. forgot to delete it.